java-checks-test-sources/src/main/files/non-compiling/checks/TextBlockToString.java

```diff
@@ -6,6 +6,8 @@
               
   hello
   
-  """.toString(); // Noncompliant@-4 [[sc=19;ec=6]] {{there's no need to call "toString()" on a text block.}}
+  """.toString(); // Noncompliant@-4 [[sc=19;ec=6;quickfixes=qf1]] {{there's no need to call "toString()" on a text block.}}
+  // fix@qf1 {{Remove "toString()"}}
+  // edit@qf1 [[sc=6;ec=17]] {{}}
 
 }
```

Comment:
We should move this entire file to the compiling sources since we now support Java 16

Reply:
Indeed :)